<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: qlineargradient(spread:pad, x1:0 y1:0, x2:1 y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
 background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 blue, stop:1 cyan);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 14pt &quot;Serif&quot;;</string>
    </property>
    <property name="title">
     <string>VERIFY</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>751</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Person Info</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(4, 4, 4);
background: transparent;</string>
      </property>
      <property name="text">
       <string>Person ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:0.5 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(4, 4, 4);
background: transparent;
</string>
      </property>
      <property name="text">
       <string>Person Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:0.5 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(4, 4, 4);
background: transparent;</string>
      </property>
      <property name="text">
       <string>Security Level</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(4, 4, 4);
background: transparent;</string>
      </property>
      <property name="text">
       <string>Cartd No</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:1 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(4, 4, 4);
background: transparent;</string>
      </property>
      <property name="text">
       <string>Date &amp; Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:1 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_Exit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color:black;</string>
      </property>
      <property name="text">
       <string>&amp;Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="l_Pic">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:transparent;
border-color: rgb(7, 7, 7);</string>
      </property>
      <property name="text">
       <string>              Pic</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:1 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:0.5 pink);
border-radius:10px;</string>
      </property>
     </widget>
     <zorder>l_Pic</zorder>
     <zorder>label</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>lineEdit_5</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_7</zorder>
     <zorder>Btn_Exit</zorder>
     <zorder>lineEdit</zorder>
     <zorder>lineEdit_2</zorder>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(4, 4, 4);
background: transparent;</string>
     </property>
     <property name="text">
      <string>RFID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background: transparent;</string>
     </property>
     <property name="text">
      <string>       Finger</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Serif&quot;;
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 white, stop:0.5 pink);
border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background: transparent;</string>
     </property>
     <property name="text">
      <string>RFID</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>91</y>
      <width>240</width>
      <height>226</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSignature"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay_Logs"/>
    <addaction name="separator"/>
    <addaction name="actionUser_Lists"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_User"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionSignature">
   <property name="text">
    <string>Signature</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionUser_Lists">
   <property name="text">
    <string>Display Users</string>
   </property>
  </action>
  <action name="actionDisplay_Logs">
   <property name="text">
    <string>Display Logs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
